ui/src/cloud/apis/demodata.ts

```diff
@@ -0,0 +1,93 @@
+// Libraries
+import {get} from 'lodash'
+import * as api from 'src/client'
+import AJAX from 'src/utils/ajax'
+
+//Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {isDemoData} from 'src/cloud/utils/filterDemoData'
+
+//Types
+import {Bucket} from 'src/types'
+import {LIMIT} from 'src/resources/constants'
+
+const baseURL = '/api/v2/experimental/sampledata'
+
+export const getDemoDataBuckets = async (): Promise<Bucket[]> => {
+  try {
+    const {data} = await AJAX({
```

Comment:
seems as though we're deprecating `AJAX` in favor of using fetch. https://github.com/influxdata/influxdb/pull/17396#discussion_r398023471. not sure if this needs to change for a demo though, your call. the fetch API seems pretty similar to the AJAX api but 🤷‍♂

Reply:
hrm... I think I'll leave it for now, since I'm not even sure this will work. I will consider switching on next iteration